nl.lostboys.rollemup.yaml

```diff
@@ -0,0 +1,73 @@
+app-id: nl.lostboys.rollemup
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: rollemup
+
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --allow=multiarch
+  - --talk-name=org.freedesktop.Notifications
+
+add-extensions:
+  org.freedesktop.Platform.Compat.i386:
+    directory: lib/i386-linux-gnu
+    version: '20.08'
+
+modules:
+  - name: platform-bootstrap
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/i386-linux-gnu
+
+  - name: xprop
+    sources:
+      - type: archive
+        url: https://xorg.freedesktop.org/archive/individual/app/xprop-1.2.5.tar.gz
+        sha256: b7bf6b6be6cf23e7966a153fc84d5901c14f01ee952fbd9d930aa48e2385d670
+
+  - name: ancient-libstdc++
```

Comment:
Any reason why this file is added directly to git instead of downloading it?

Reply:
Because it's just one simple file. There is no reason to rely on external resources when I can just put it here. It's still FOSS.

Reply:
Mostly because git is not supposed to be used for storing git files. To turn your question around, why don't you also commit tarballs listed as `archive`?

Reply:
> why don't you also commit tarballs listed as archive?

Challenge accepted: https://github.com/TomaszGasior/mushrooms-rpms/tree/3ee7e138245d79f88074e89f6a63d4ee725d832b :P 

But yeah, I understand your point of view. Fixed.